DisplayI2C1.cs

```diff
@@ -0,0 +1,274 @@
+﻿/**
+ *  Character-LCD-over-I2C 
+ *  ===================
+ *  Connect HD44780 LCD character display to Windows 10 IoT devices via I2C and PCF8574
+ *
+ *  Author: Jaroslav Zivny
+ *  Version: 1.1
+ *  Keywords: Windows IoT, LCD, HD44780, PCF8574, I2C bus, Raspberry Pi 2
+ *  Git: https://github.com/DzeryCZ/Character-LCD-over-I2C
+**/
+
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Windows.Devices.Gpio;
+using Windows.Devices.Enumeration;
+using Windows.Devices.I2c;
+using System.Diagnostics;
+
+namespace displayI2C
```

Comment:
It seems, that `namespace` should have the first letter Uppercase as well. 
https://en.wikibooks.org/wiki/C_Sharp_Programming/Naming

Reply:
This is something I left for you, the namespace should be something like the library name, and not the class name you'll be using. So something like "CharacterLcdOverI2C" or similar, then if we end up making a nuget package you can just do "using CharacterLcdOverI2C;" at the top of your file and then use DisplayI2C in your code.

Reply:
Yep, let's rename it to `CharacterLcdOverI2C`, it seems to be a good name.